### DISCUSSION TITLE
Retiring as maintainer

### DISCUSSION THREAD

**Discussion by arjun024** [paketo-buildpacks/feedback#125] [Node.js Team]
Hi @paketo-buildpacks/nodejs-maintainers 
I'd like to announce I'm retiring from my role as maintainer of the Nodejs subteam. I'd like to continue in the subteam as a contributor. Thank you for having me.

cc @paketo-buildpacks/steering-committee

**Comment by ryanmoran** [comment]
Thank you for your service! 🎉
